product_docs/docs/net_connector/8.0.2.1/07_retrieving_database_records.mdx

```diff
@@ -62,7 +62,7 @@ private void Page_Load(object sender, System.EventArgs e)
 
 ```
 
```

Comment:
Is there an encoding error? The text that follows is appearing in blue.

Reply:
yeah, the code block is cpp